networkx/algorithms/traversal/breadth_first_search.py

```diff
@@ -383,28 +383,37 @@ def descendants_at_distance(G, source, distance):
     -------
     set()
         The descendants of `source` in `G` at the given `distance` from `source`
+
+    Examples
+    --------
+    >>> G = nx.path_graph(3)
```

Comment:
Maybe make this path length longer so it is clear what distance means in this context.

Reply:
Enlarged it slightly and changed the source to a node which is not an endpoint of the path, but happy to change it further if needed.